src/experimental/patterns/ec2-app.ts

```diff
@@ -274,7 +289,8 @@ export class GuRollingUpdatePolicyExperimental {
       If AWS ever supports suspending scale-out and scale-in independently, we should allow scale-out.
        */
       suspendProcesses: [ScalingProcess.ALARM_NOTIFICATION],
-      // Note: there is also an important property called pauseTime, but this is set via an Aspect.
+      // Note: this is increased via an Aspect which also takes healthcheck grace period into account.
+      pauseTime: slowStartDuration ?? Duration.seconds(0),
```

Comment:
minor: Would having a default value in the function signature highlight it more/be more maintainable?

```ts
getPolicy(maximumInstances: number, minimumInstances?: number, slowStarDuration: Duration = Duration.seconds(0)) {
  ...
}
```